block_util/src/raw_async.rs

```diff
@@ -11,6 +11,7 @@ use std::io::{Seek, SeekFrom};
 use std::os::unix::io::{AsRawFd, RawFd};
 use vmm_sys_util::eventfd::EventFd;
 
+/// Enabled with the `"io-uring"` feature
```

Comment:
I don't think this is necessary. The crate top-level `lib.rs` already contains this information (with putting the whole `raw_async` mod behind the `io-uring` feature gate).